guides/contributing_to_the_handbook.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to the Handbook
+
+This guide aims to make it easier to submit changes to the Handbook without using `git` or a text editor.
+
+## Adding new pages in Github
+
+### Creating the page
```

Comment:
Maybe could do with something around editing an existing file as well?

Reply:
Out of scope! Good point though.